src/strings/en-us.json

```diff
@@ -1123,7 +1123,7 @@
     "LabelMetadataPath": "Metadata path:",
     "LabelMetadataPathHelp": "Specify a custom location for downloaded artwork and metadata.",
     "LabelWebPath": "Web path:",
-    "LabelWebPathHelp": "The path where the web client source is located.",
+    "LabelWebPathHelp": "The path where the web client source is located. Do not change this unless you plan on moving the web files, or the web interface will break.",
```

Comment:
I would rather we _not_ let users edit this via web UI (warning text is something no one would actually read).